packages/kbn-code-owners/src/file_code_owner.ts

```diff
@@ -66,3 +69,32 @@ export function getCodeOwnersForFile(
 
   return match?.teams;
 }
+
+/**
+ * Run the getCodeOwnersForFile() method above.
+ * Report back to the cli with either success and the owner(s), or a failure.
+ *
+ * This function depends on a --file param being passed on the cli, like this:
+ * $ node scripts/get_owners_for_file.js --file SOME-FILE
+ */
+export async function runGetOwnersForFileCli() {
+  run(
+    async ({ flags, log }) => {
+      const targetFile = flags.file as string;
+      if (!targetFile) throw createFlagError(`Missing --flag argument`);
+      existOrThrow(targetFile); // This call is duplicated in getPathsWithOwnersReversed(), so this is a short circuit
+      const result = getCodeOwnersForFile(targetFile);
+      if (result) log.success(result);
+      else log.error(`Ownership of file [${targetFile}] is UNKNOWN`);
+    },
+    {
+      description: 'Report file ownership from GitHub CODEOWNERS file.',
+      flags: {
+        string: ['file'],
```

Comment:
@delanni is this not it bud?

Reply:
@delanni Haven't heard from ya man.  Gonna merge this soon.  Please lemme know if you've an objection

Reply:
Sorry, I didn't see the message.
I meant the wording of the error:
```
throw createFlagError(`Missing --flag argument`);
```

but there's no `--flag` argument, it's `--file`

Reply:
Ahhhhh!!!!

Reply:
https://github.com/elastic/kibana/pull/193581

@delanni